bedrock/firefox/templates/firefox/new/scene2.html

```diff
@@ -34,6 +34,11 @@ <h1>{{ _('Let’s do this!') }}</h1>
   {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
 </div>
 
+<p class="windows-xpvista-download">
+  <span class="icon" aria-hidden="true"></span>
+  {{ _('<a href="%(url)s">Mozilla no longer provides security updates for Firefox on Windows XP or Vista</a>, but you can still download the final Windows 32-bit version.')|format(url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}
```

Comment:
We're about to land a new default template for scene2 in #6092. Once that merges, we should rebase and make sure the message is visible on both templates.

Reply:
I also wonder if the copy here might be confusing still. The user has already initiated the download, do they need to download something else, where is the 32bit version?

Perhaps the existing string from scene1 would make more sense here?

Reply:
Okay, adding the Blocked label then.

Reply:
@kyoshino please see my suggestion on using the string from scene1 here instead.

Reply:
@kyoshino I don't think this issue needs to be blocked here, we can reuse the existing strings we have already I think.